x-pack/plugins/security_solution/server/lib/detection_engine/rules/patch_rules.test.ts

```diff
@@ -141,25 +131,23 @@ describe('patchRules', () => {
     });
 
     it('does not update actions if none are specified', async () => {
-      const ruleOptions = getPatchRulesOptionsMock();
-      delete ruleOptions.actions;
-      if (ruleOptions.rule != null) {
-        ruleOptions.rule.actions = [
-          {
-            actionTypeId: '.slack',
-            id: '2933e581-d81c-4fe3-88fe-c57c6b8a5bfd',
-            params: {
-              message: 'Rule {{context.rule.name}} generated {{state.signals_count}} signals',
-            },
-            group: 'default',
+      const rulesClient = rulesClientMock.create();
+      const params = getCreateRulesSchemaMock();
```

Comment:
Nit, but I think `{ actions, ...params }` is preferable to `delete` here because it doesn't mutate the returned object.